upstream.wrap

```diff
@@ -0,0 +1,9 @@
+[wrap-file]
+directory = spdlog-1.8.3
+
+source_url = https://github.com/gabime/spdlog/archive/refs/tags/v1.8.3.tar.gz
```

Comment:
What's the intention of refs/tags/ here? It works with

```diff
-source_url = https://github.com/gabime/spdlog/archive/refs/tags/v1.8.3.tar.gz
+source_url = https://github.com/gabime/spdlog/archive/v1.8.3.tar.gz
```

And typically I see the latter.